services/bots/tests/test_helpers.py

```diff
@@ -0,0 +1,116 @@
+from unittest.mock import patch
+
+from shared.torngit.base import TokenType
+from shared.typings.oauth_token_types import Token
+
+from services.bots.helpers import (
+    get_dedicated_app_token_from_config,
+    get_token_type_from_config,
+)
+
+
+@patch("services.bots.helpers.get_github_integration_token")
+def test_get_dedicated_app_token_from_config(
+    mock_get_integration_token, mock_configuration
+):
+    mock_configuration.set_params(
+        {
+            "github": {
```

Comment:
This looks almost duplicated from the params set in the next test. Can we turn this dict into a fixture? ([docs for fixtures](https://docs.pytest.org/en/7.1.x/explanation/fixtures.html#about-fixtures))